### DISCUSSION TITLE
Add floor temperature sensor (and logic)

### DISCUSSION THREAD

**Discussion by pixide** [ScratMan/HASmartThermostat#90] [Ideas]
Thanks for the work. Please could you add another sensor to the thermostat configuration that would receive the value of the floor temperature sensor. For floor heating (electrical, water) it is said the surface of the floor should not exceed certain maximal temperature (about 29 C) for human feet comfort. It is also mandatory (in some countries?) to install these floor temperature sensors under the floor surface while installing the floor heating. The hardware thermostats then work with them and ensure that the floor surface temperature is never exceeded.

**Comment by HuismAndre** [comment]
With water based floor heating, the system itself should have a protection against the incoming water being too hot. In all water based systems I've seen they use a thermostatic valve that shuts off the water supply when it reaches a certain temperature (40 deg C in my case). With water being 40 deg C at most, the floor never gets hotter than 30 deg C.
